truffle-config.js

```diff
@@ -0,0 +1,152 @@
+require("dotenv").config();
+const HDWalletProvider = require("@truffle/hdwallet-provider");
+
+const CONSISTENT_DEPOLY_ADDR = "0xB3c12C9535626cfFef5FE76d5A4e198A5e753FD9";
+// const CONSISTENT_DEPOLY_ADDR = "0x70ebDF2a1C9C7A45141760419c42056E3DDfA690";
+// const CONSISTENT_DEPOLY_ADDR ="0x1Dde3A77975235E6DdB31389c3D599dE67AbA1Ad";
```

Comment:
what are these addresses? remove or add comment for the purpose, and remove if they are irrelevant